tsdb/importer/import.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package importer
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"math"
+	"os"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/pkg/errors"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/pkg/textparse"
+	"github.com/prometheus/prometheus/tsdb"
+	tsdb_errors "github.com/prometheus/prometheus/tsdb/errors"
+	"github.com/prometheus/prometheus/tsdb/importer/openmetrics"
+)
+
+var merr tsdb_errors.MultiError
+
+func blockMaxTimeForTheTimestamp(t int64, width int64) (maxt int64) {
+	return (t/width)*width + width
+}
+
+// MinInt64 functions returns the minimum value between two 64bit integers
+func MinInt64(x, y int64) int64 {
+	if x < y {
+		return x
+	}
+	return y
+}
+
+// MaxInt64 functions returns the maximum value between two 64bit integers
+func MaxInt64(x, y int64) int64 {
+	if x > y {
+		return x
+	}
+	return y
+}
+
+// Import function reads an OM file, creates two hour blocks and writes that to the TSDB.
+func Import(path string, outputDir string, DefaultBlockDuration int64) (err error) {
```

Comment:
Same comment as above
```suggestion
func Import(path string, outputDir string, blockDuration int64) (err error) {
```